lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -1,6 +1,7 @@
 require 'base64'
 require 'time'
 require 'cgi'
+require 'active_support/core_ext/hash/except'
```

Comment:
The gemspec already has `s.add_dependency('activesupport', '>= 3.2.9')`. Is anything more necessary?